docs/mutation-data-transcript-annotation.md

```diff
@@ -0,0 +1,85 @@
+website for the [BRAF gene](https://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000157764;r=7:140719327-140924929#): 
+
+<img width="995" alt="Screen Shot 2022-10-05 at 9 35 55 AM" src="https://user-images.githubusercontent.com/1334004/194073821-9a43cab2-3d31-40ab-b47e-517f1ce8bca3.png">
+
+The transcript `ENST00000288602.6` is 2480 base pairs long (nucleotides ACGT) and the associated protein isoform is 766 
+amino acids (V/P/etc). You can see we are showing that same transcript and protein isoform on [cBioPortal](https://bit.ly/3vaclXN): 
+
+<img width="1052" alt="Screen Shot 2022-10-05 at 9 37 32 AM" src="https://user-images.githubusercontent.com/1334004/194074248-c3e205b4-c2e4-4e19-a541-17517dc07913.png"> 
+
+For each gene name in cBioPortal a canonical/default transcript is assigned. These assignments are stored in [Genome 
+Nexus](https://www.genomenexus.org/) and explained below. Although cBioPortal does not store changes to different 
+transcripts/isoforms for each mutation in the database itself, it does allow viewing them on the [Mutations Tab](https://bit.ly/39hVtDd) by re-annotating the mutations on the fly through [Genome Nexus](https://www.genomenexus.org/) 
+whenever a user clicks on the transcript dropdown. 
+
+### Transcript Assignment 
+
+The cBioPortal database stores one gene + protein change annotation for each mutation event in the database. To allow 
+comparing mutation data across studies it is important to annotate the mutation data (be it in MAF or VCF format) in 
+the same way, otherwise the gene + protein changes can mean entirely different things. For all public studies stored 
+in [datahub](https://github.com/cBioPortal/datahub/tree/master/public) we leverage [Genome Nexus](https:
+//www.genomenexus.org) to do so. Genome Nexus assigns one canonical Ensembl Transcript + gene name + protein change for 
+each mutation. You can find the mapping of hugo symbol to transcript id[here](https://github.com/genome-nexus/genome-
+nexus-importer/blob/master/data/grch37_ensembl92/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt).There are 
+two sets of default transcripts: `uniprot` and `mskcc`. We recommend to use the `mskcc` set of transcripts when 
+starting from scratch, since these are more up to date and correspond to transcripts that were chosen as relevant for 
+clinical sequencing at MSKCC. The `uniprot` set of transcripts was constructed several years ago, but we are no longer 
+certain about the logic on how to reconstruct them hence they are not being kept up to date. One can see the 
+differences between the two in [this file](https://github.com/cBioPortal/cbioportal-
+frontend/files/9498680/genes_with_different_uniprot_mskcc_isoforms.txt). For the public cBioPortal (https:
+//www.cbioportal.org) we are using `mskcc`, for for the GENIE cBioPortal (https://genie.cbioportal.org) we use `uniprot`
```

Comment:
why is GENIE using this? That sounds like a mistake in the context of previous paragraph.

Reply:
just legacy reasons - we are trying to upgrade to MSKCC but will take a bit